lib/bridge/MatrixHandler.js

```diff
@@ -369,7 +368,92 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         return;
     }
     else {
-        req.log.info("No valid admin command: %s", event.content.body);
+        let fmt = '\nirc.server\n'+
+                    '(COMMAND [arg0 [arg1 [...]]])\n'+
+                    '(COMMAND [arg0 [arg1 [...]]])\n'+
+                    'irc.server2';
+        req.log.info(`No valid (old form) admin command, will try new format`);
+
+        // Assumes commands have the form
+        // irc.server
+        // (COMMAND [arg0 [arg1 [...]]])
+        // (COMMAND [arg0 [arg1 [...]]])
+        // irc.server2
+        // ...
+        let commandBlocks = event.content.body;
+
+        let currentServer = ircServer;
+        let blacklist = ['PROTOCTL'];
+
+        try {
+            let lines = commandBlocks.replace(/\r/, "").split("\n");
```

Comment:
If you looked carefully at my comment, you would have see the little `g` at the end there. That is the [global flag](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/RegExp) - you need it in this case:

```javascript
> s = "foodfoodfood"
'foodfoodfood'
> s.replace(/f/, "")
'oodfoodfood'
> s.replace(/f/g, "")
'oodoodood'
```